lib/Doctrine/DBAL/Driver/PDOConnection.php

```diff
@@ -115,4 +117,44 @@ protected function createStatement(\PDOStatement $stmt) : PDOStatement
     {
         return new PDOStatement($stmt);
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function beginTransaction()
+    {
+        return $this->conn->beginTransaction();
```

Comment:
Not wrapping this into a driver exception. Could be resolved in a different ticket.